plugins/oracle_ckn_daemon/tests/test_oracle_daemon.py

```diff
@@ -0,0 +1,53 @@
+import unittest
```

Comment:
Docstrings are generally good specially for plugins since it will serve as a template for others to write theirs